docs/concepts/workloads/containers/container-environment.md

```diff
@@ -0,0 +1,63 @@
+assignees:
+- mikedanese
+- thockin
+title: Container Environment
+---
+
+{% capture overview %}
+This page describes the resources available to containers in the Container Environment. 
+{% endcapture %}
+
+{:toc}
+
+{% capture body %}
+## Container environment
+
+The Kubernetes Container Environment provides several important resources to containers:
```

Comment:
I don't think Container Environment is the proper name of anything in Kubernetes. So no caps.